Misc/NEWS.d/next/Library/2019-10-10-16-53-00.bpo-38431.d5wzNp.rst

```diff
@@ -0,0 +1,2 @@
+Fix ``__repr__`` method for ``dataclasses.InitVar`` to work with typing
+objects.
```

Comment:
Please add "Patch by ...".

Reply:
I assume these comments also apply to #16702 

I'm not sure what you mean here, could you elaborate?

Reply:
Adding "patch by <full name>" for credit since this is added to changelog.

Reply:
done.

Reply:
Sorry, it's "patch by full name." as additional sentence for credit. GitHub removed my content with tags <>

Reply:
wait, you mean `patched by: <Samuel Colvin>`?

Or perhaps you're confused since you're commenting on the wrong PR? should be #16702

Reply:
I'm just a bit confused since that format isn't used in any other news entries as far as I can tell.

Reply:
I answered here since it had the question initially :) The news entry would be as below for other PR. Sorry for the confusion due to markdown rendering.

```
Fix ``__repr__`` method for ``dataclasses.InitVar`` to work with typing
objects. Patch by Samuel Colvin.
```

Reply:
okay, that's pretty much what I have in the other PR. Is there still something to change?

Reply:
Nothing else, I added the verbatim version since patch by would be a separate sentence and not in past tense as above for most cases. But I will leave it to the core dev who is merging to take care of wording and not pedantic about it. Thanks for the changes.

https://devguide.python.org/committing/#handling-others-code and has an example of the format widely used.

```
Third, ensure the patch is attributed correctly with the contributor’s name in 
Misc/ACKS if they aren’t already there (and didn’t add themselves in their patch) 
and by mentioning “Patch by <x>” in the Misc/NEWS.d entry and the check-in 
message. If the patch has been heavily modified then “Initial patch by <x>” is 
an appropriate alternate wording. GitHub now supports multiple authors in a commit. 
Add Co-authored-by: name <name@example.com> at the end of the commit message.
```